index.html

```diff
@@ -95,7 +95,7 @@ <h2 class="project-titles">Tonic</h2>
               <li class="tags">css</li>
               <li class="tags">javascripts</li>
             </ul>
-            <button class="btn" type="button">See Project</button>
+            <button class="btn btn-projects" type="button" id="1">See Project</button>
```

Comment:
- [ ] You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads. (Please make sure the entire project section is created using javascript dynamically, You may want to remove the project section from the HTML file and create it using javascript).

Reply:
- [ ] Kindly ensure all your projects are loaded dynamically, currently, you have hardcoded all the projects in your HTML file. kindly create an array in your profile.js file that contains all the projects.